samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,139 @@
+.. _gps_with_supl_support_sample:
+
+nRF9160: GPS sockets and SUPL client library
+############################################
+
+The GPS sample demonstrates how to retrieve `GPS`_ data.
+If Secure User-Plane Location (SUPL) support is enabled, it also shows how to improve the GPS fix accuracy and fix speed with `A-GPS`_ data from a SUPL server.
+See :ref:`supl_client` for information on enabling SUPL support for the sample.
+The sample first creates a `GNSS`_ socket.
```

Comment:
Let's start a new paragraph here (before "The sample first ...").

Reply:
Modified